### PR TITLE
Update add-staff.md

### PR DESCRIPTION
**Pull request by msbemba**
tested and found that the add staff option is not available in the new bookings experience.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6453

File: microsoft-365/bookings/add-staff.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Although Bookings is a feature of Microsoft 365, not all of your staff members a
 
 ## Steps
 
+> [!NOTE]
+> The steps are not yet available on the new bookings experience.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/add-staff.md]:
```suggestion
> These steps are not yet available in the new Bookings experience.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**